projects/wave-core-new/src/lib/util/conversions.ts

```diff
@@ -25,3 +27,37 @@ export const bboxDictToExtent = (extent: BBoxDict): [number, number, number, num
  * Convert a unix timestamp in ms to an ISO timestamp string.
  */
 export const unixTimestampToIsoString = (unixTimestamp: number): string => utc(unixTimestamp).toISOString();
+
+// TODO: use a faster hash function
+export const featureToHash = (feature: OlFeature): number => {
+    const hashFn = function (str: string): number {
+        let hash = 0;
+        let i;
+        let chr;
+        if (str.length === 0) return hash;
+        for (i = 0; i < str.length; i++) {
+            chr = str.charCodeAt(i);
+            // eslint-disable-next-line no-bitwise
+            hash = (hash << 5) - hash + chr;
+            // eslint-disable-next-line no-bitwise
+            hash |= 0; // Convert to 32bit integer
+        }
+        return hash;
+    };
```

Comment:
We could use just one hash implementation.

https://github.com/geo-engine/geoengine-ui/blob/5990a2e228624cd4cc9e36e3d10fe6a662d07376/projects/wave-core-new/src/lib/util/icons.ts#L5-L15

Reply:
2da2118